input/templates/my-account-wishlist.hbs

```diff
@@ -1,72 +1,71 @@
+                                                <form id="form-remove-from-wishlist{{index}}" name="remove-from-wishlist"
+                                                      action="{{@root.meta._links.removeFromWishlist.href}}" method="post">
+                                                    <input type="hidden" name="csrfToken"
+                                                           value="{{@root.meta.csrfToken}}"/>
+                                                    <input type="hidden" name="lineItemId" value="{{product.lineItemId}}">
+
+                                                    <div class="my-account-wishlist-remove-btn">
+                                                        <button type="submit">{{i18n
+                                                                "catalog:removeBtn.title"}}</button>
+                                                    </div>
+                                                </form>
+                                            </div>
+                                        {{/each}}
+                                    </div>
+                                </div>
+                                <div class="my-account-wishlist-pagination-bottom">
+                                </div>
+                                <div class="my-account-wishlist-clear-btn">
+                                    <form id="form-clear-wishlist" name="clear-wishlist"
+                                          action="{{@root.meta._links.clearWishlist.href}}" method="post">
+                                        <input type="hidden" name="csrfToken"
+                                               value="{{@root.meta.csrfToken}}"/>
+
+                                        <button type="submit">{{i18n "my-account:myWishlist.clear"}}</button>
+                                    </form>
+                                </div>
+                            </div>
                         </div>
-                      {{/each}}
-                    </div>
-                  </div>
-                  <div class="my-account-wishlist-pagination-bottom">
-                    <div class="row">
-                      <div class="col-sm-12 hidden-xs text-center custom-pagination">
-                        <ul class="page-numbers">
-                          {{> common/pagination pagination=content.pagination}}
-                        </ul>
-                      </div>
                     </div>
-                  </div>
-                  <div class="my-account-wishlist-clear-btn">
-                    <button>{{content.clearWishlistBtn}}</button>
-                  </div>
                 </div>
-              </div>
             </div>
-          </div>
         </div>
-      </div>
     </div>
-  </div>
-  {{!-- {{> common/wishlist}} --}}
-  {{> common/footer}}
-  {{> common/htmlscripts}}
+</div>
+{{!-- {{> common/wishlist}} --}}
```

Comment:
You can remove definitely that if we don't use it (which makes sense, since we are already seeing the whole wishlist, we don't need the mini one 😅 ).